Source/WebInspectorUI/UserInterface/Models/CSSKeywordCompletions.js

```diff
@@ -1317,6 +1320,15 @@ WI.CSSKeywordCompletions._propertyKeywordMap = {
     "resize": [
         "none", "auto", "both", "horizontal", "vertical",
     ],
+    "ruby-align": [
+        "start", "center", "space-between", "space-around",
```

Comment:
are these not provided to the frontend by `CSS.getSupportedCSSProperties`?

ever since [204647@main](https://commits.webkit.org/204647@main) the backend should be including as much of this kind of info as possible, with the hardcoded list in `WI.CSSKeywordCompletions` being more of a legacy thing, so if these are included in `CSS.getSupportedCSSProperties` then id prefer we not add them as it's unnecessary